addons/mail/static/src/owl/components/composer_text_input/composer_text_input.js

```diff
@@ -120,16 +204,50 @@ class ComposerTextInput extends Component {
         return this.getContent() === "";
     }
 
+    /**
+     * Triggers RPC to search mentioned partners based on user's research
+     * and add im_status to resulted partners.
+     *
+     * @private
+     * @param model
+     * @param method
+     * @param kwargs
+     */
+    async _mentionFetchThrottled(model, method, kwargs) {
+        // Prequel of mentions of relevant partners
+        // const suggestions = this.storeGetters.mentionSuggestions(this._mentionWord);
+        // const prefetchedPartners = await this.env.rpc({
+        //     model: 'mail.channel',
+        //     method: 'channel_fetch_listeners',
+        //     args: [this.storeProps.thread.uuid],
+        //     }, {
+        //         shadow: true
+        // });
+
+        this.state.mentionSuggestions = await this.env.rpc({
+            model: model,
+            method: method,
+            kwargs: kwargs,
+        });
+
+        for (const mentionSuggestion of this.state.mentionSuggestions[0]) {
+            mentionSuggestion.im_status = this.env.call('mail_service', 'getImStatus', { partnerID: mentionSuggestion.id });
```

Comment:
You should no longer rely on `mail_service`. This is the old store, and the intent is to drop it after our refractoring.